.github/workflows/unit_tests.yml

```diff
@@ -66,6 +66,25 @@ jobs:
           command: test
           args: --lib
 
+      - name: Setup Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.8'
+
+      - name: Install pylint and pycodestyle
+        run: |
+          python -m pip install --upgrade pip
+          pip install pylint pycodestyle
+
+      - name: Run pylint and pycodestyle
+        run: |
+          find . -type f -name "*.py" | xargs pylint \
```

Comment:
we probably want to run pylint per python project: for example `pylint ./local-tests` should be enough to run for all python files in the `local-tests` project (and is much simpler).

Reply:
done